cloudify_cli/commands/executions.py

```diff
@@ -265,7 +271,9 @@ def manager_start(workflow_id,
                         ' start when possible.')
             return
         if execution.status == 'scheduled':
-            logger.info('Execution is scheduled for {0}.'.format(schedule))
+            logger.info('Execution is scheduled for {0}.\nYou can see the '
+                        'execution schedule using `cfy deployments schedule '
+                        'list`.'.format(schedule))
```

Comment:
aww this the only place you used logger+str.format :P (yes I know it was like this before)